cmd/create.go

```diff
@@ -277,6 +284,17 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 				"firewall-port": inletsPort,
 			},
 		}, nil
+	} else if provider == "ec2" {
+		return &provision.BasicHost{
+			Name:     name,
+			OS:       "ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-20191114",
+			Plan:     "t3.nano",
```

Comment:
How does this differ to the t2.nano? What made this the better option?

Reply:
t3 is better performing and slightly cheaper.

Reply:
Great 👍 let’s see what people think to that choice, most have been fine with the choices made so far, but we’ll see once it’s out there.